docs/simulation/security/contingency-dsl.md

```diff
@@ -70,62 +89,71 @@ for (l in network.lines) {
 }
 ```
 
-In the following example, we use the [Stream API](https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/util/stream/package-summary.html) to create a list of contingencies with the 3 first 225 kV french lines:  
+In the following example, we use
+the [Stream API](https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/util/stream/package-summary.html) to
+create a list of contingencies with the 3 first 225 kV french lines:
+
 ```groovy
 import com.powsybl.iidm.network.Country
 
 network.lineStream
-    .filter({l -> l.terminal1.voltageLevel.substation.country == Country.FR})
-    .filter({l -> l.terminal2.voltageLevel.substation.country == Country.FR})
-    .filter({l -> l.terminal1.voltageLevel.nominalV == 225})
-    .filter({l -> l.terminal2.voltageLevel.nominalV == 225})
-    .limit(3)
-    .forEach({l ->
-        contingency(l.id) {
-            equipments l.id
-        }
-    })
+        .filter({ l -> l.terminal1.voltageLevel.substation.country == Country.FR })
+        .filter({ l -> l.terminal2.voltageLevel.substation.country == Country.FR })
+        .filter({ l -> l.terminal1.voltageLevel.nominalV == 225 })
+        .filter({ l -> l.terminal2.voltageLevel.nominalV == 225 })
+        .limit(3)
+        .forEach({ l ->
+            contingency(l.id) {
+                equipments l.id
+            }
+        })
```

Comment:
Those added tabulations are not mandatory, it's a personal choice to set the settings to 4 or 8 spaces